.github/workflows/prerelease_canaries.yml

```diff
@@ -0,0 +1,88 @@
+name: 🐤 Canaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to test'
+        required: true
+      linux:
+        description: '🐧 Should create canaries for Linux?'
+        type: boolean
+        default: true
+        required: false
+      macos:
+        description: '🍏 Should create canaries for MacOS?'
+        type: boolean
+        default: true
+        required: false
+      windows:
+        description: '🪟 Should create canaries for Windows?'
+        type: boolean
+        default: true
+        required: false
+      windows:
```

Comment:
```suggestion
      prune:
```
I think this input name should be prune

Reply:
nice 👀  thanks!

Reply:
both fixed in https://github.com/newrelic/infrastructure-agent/pull/1531/commits/6c5463892f4ab7a5888352fac804522c9795723f